Steg2

ingheden opened the pull request:
Lagt till interaktioner samt gjort större ändring i koden då den första iterationen var svår att bygga på.

ingheden committed:
Steg2

Lagt till interaktioner samt gjort större ändring i koden då den första iterationen var svår att bygga på.

mogge1997 reviewed (approved):
Jag tycker du har gjort det bra med de olika delarna då är klar med. Som du också är medveten om saknas filtreringsfunktionen, vilket jag säkert tror du kommer och kunna lösa. I sin helhet tycker jag att du har gjort ett bra jobb, det saknas enbart filtreringen.

ingheden committed:
Fixat till

ingheden committed:
rework

Pga dåligt följda intstruktioner behövs koden ändras mer eller mindre helt för att få API att funka, dumt att freebasea!
